frameworks/g-cloud-9/messages/contract_variation_1.yml

```diff
@@ -18,7 +18,7 @@ variation_comes_in_effect_description: We’ll tell you when CCS has countersign
 
 variation_agreement_label: I have the legal authority to accept these changes and agree that the company will be bound by them.
 
-confirmation_message: You have accepted the proposed changes. They will come into effect when CCS has countersigned them. We’ll let you know when this happens.
+confirmation_message: You've accepted the proposed changes. They'll come into effect when CCS has countersigned them. We’ll let you know when this happens.
```

Comment:
These should be fancy apostrophes (`’`). Copy-pasted from the `CCS may amend this [...]` bit below. We (try to) use these rather than bog standard apostrophes because content prefers the look of them.